pennylane/pauli/utils.py

```diff
@@ -273,14 +305,14 @@ def pauli_to_binary(pauli_word, n_qubits=None, wire_map=None, check_is_pauli_wor
     >>> pauli_to_binary(qml.PauliX(0) @ qml.PauliX(5),  wire_map=wire_map)
     array([1., 0., 0., 0., 0., 1., 0., 0., 0., 0., 0., 0.])
     """
+    wire_map = wire_map or {w: i for i, w in enumerate(pauli_word.wires)}
+
+    if pauli_word.pauli_rep:
```

Comment:
Doesnt look like `pauli_to_binary` would work on PauliWords. I am getting
```
>>> pw = PauliWord({0:"X"})
>>> pw.pauli_rep
AttributeError: 'PauliWord' object has no attribute 'pauli_rep'
```

Confirmed by trying `qml.pauli.pauli_to_binary(pw)`

Reply:
The function isn't supposed to work for `PauliWord`s. The docstring says that the type of the `pauli_word` argument is `Union[Identity, PauliX, PauliY, PauliZ, Tensor, Prod, SProd]`, so it only works for `Operator`s. The name is definitely misleading due to ambiguity between the class `PauliWord` and operators that are valid Pauli words.

Reply:
Ah right. Maybe something to add in the future so you could use that e.g. on a list of mixed operators (that contains paulis and ops). Not necessary for this PR though.